Remotion/Validation/UnitTests/Validators/LessThanValidatorTest.cs

```diff
@@ -61,15 +63,29 @@ public void Validate_WithPropertyValueGreaterThanComparisonValue_ReturnsSingleVa
       var validationFailures = validator.Validate(propertyValidatorContext).ToArray();
 
       Assert.That(validationFailures.Length, Is.EqualTo(1));
-      //TODO RM-5906: Assert ValidatedObject, ValidatedProperty, ValidatedValue
+      Assert.That(validationFailures[0].ValidatedObject, Is.EqualTo(propertyValidatorContext.Instance));
+      Assert.That(validationFailures[0].ValidatedProperty, Is.EqualTo(propertyValidatorContext.Property));
+      Assert.That(validationFailures[0].ValidatedPropertyValue, Is.EqualTo(propertyValidatorContext.PropertyValue));
       Assert.That(validationFailures[0].ErrorMessage, Is.EqualTo("The value must be less than '3'."));
       Assert.That(validationFailures[0].LocalizedValidationMessage, Is.EqualTo("Custom validation message: '3'."));
     }
 
     [Test]
-    [Ignore("RM-5906")]
     public void Validate_WithIComparable_CallsCompareTo ()
     {
+      var comparisonValueMock = new Mock<IComparable>(MockBehavior.Strict);
+      var propertyValueMock = new Mock<IComparable>(MockBehavior.Strict);
+      propertyValueMock.Setup(_ => _.CompareTo(comparisonValueMock.Object))
+          .Returns(-1)
+          .Verifiable();
```

Comment:
Formatting:
when we write multi-line mock statements, each part is on a new line, inluding the Setup call. That way, we can read it in visual alignment. Historically (with RhinoMocks), the Setup-call was actuall either `Stub` or `Expect` so the alignment was even more imporant to quickly understand the test.